FirstProject/feedback.html

```diff
@@ -0,0 +1,87 @@
+               </div>
+           </form>
+        </header>
+
+    <div class="row">
+        <div class="container">
+            <div class="col-md-12">
+                <div class="col-md-6">
+                    <img class="left_image" src="img/new%20feedback.jpg"/>
+                </div>
+                <div class="col-md-6">
+                    <h3 class="right_description" style="margin-top:50px;">Submit new feedback</h3>
+                    <p style="font-size: 13px; margin-top: 60px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra
+                        nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus,
+                        tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque
+                        nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in
+                        tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis
+                        sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit
+                        amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque
+                        arcu mauris, malesuada quis ornare accumsan, blandit sed diam.
+                    </p>
+                </div>
+            </div>
+
+            <div class="col-md-12" style="margin-top:150px;">
+                <div class="col-md-6">
+                    <h3 class="right_description">View feedback results</h3>
+                    <p style="font-size: 13px; margin-top: 60px;">Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla quam velit, vulputate eu pharetra
+                        nec, mattis ac neque. Duis vulputate commodo lectus, ac blandit elit tincidunt id. Sed rhoncus,
+                        tortor sed eleifend tristique, tortor mauris molestie elit, et lacinia ipsum quam nec dui. Quisque
+                        nec mauris sit amet elit iaculis pretium sit amet quis magna. Aenean velit odio, elementum in
+                        tempus ut, vehicula eu diam. Pellentesque rhoncus aliquam mattis. Ut vulputate eros sed felis
+                        sodales nec vulputate justo hendrerit. Vivamus varius pretium ligula, a aliquam odio euismod sit
+                        amet. Quisque laoreet sem sit amet orci ullamcorper at ultricies metus viverra. Pellentesque
+                        arcu mauris, malesuada quis ornare accumsan, blandit sed diam.
+                    </p>
+                </div>
+                <div class="col-md-6">
+                    <img class="left_image" style="float:right;" src="img/results.jpg"/>
+                </div>
+            </div>
+        </div>
+        <footer class="container-fluid" style="margin-top: 80px;">
+            <div class="col-md-4">
```

Comment:
Are these empty containers required? If not, are there any options available to not include them in the markup?